src/views/AmalgamationSelection.vue

```diff
@@ -171,20 +175,20 @@ export default class AmalgamationSelection extends Vue {
 
   /** Start Regular Long-form button pressed. */
   async startRegularAmalgamation (): Promise<any> {
-    // let legalType = null
-    // if (this.isBComp) legalType = CorpTypeCd.BC_COMPANY
-    // else legalType = this.getLegalType
-
-    window.alert('This action is not available at the moment. Please check again later.')
+    let legalType = null
+    if (this.isBComp) legalType = CorpTypeCd.BC_COMPANY
```

Comment:
Why do this? Please add code comment.

Reply:
I decided to remove this as I think I misunderstood the requirement Sev.

Now, we're creating an amalgamation application for a benefit company normally if the company we're on is a benefit one.

Reply:
OK! Resolving.